packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -44,6 +50,23 @@ import 'theme_data.dart';
 /// contrast ratio with their matching colors of at least 4.5:1 in order to
 /// be readable.
 ///
+/// ### Setting Colors in Flutter
+///
+/// Flutter's Material widgets can be assigned colors at the widget level
+/// using widget properties,
+/// or at the app level using theme classes.
+///
+/// For example, you can set the background of the [AppBar] by
+/// setting the [AppBar.backgroundColor] to a specific [Color] value.
+///
+/// To globally set the AppBar background color for your app, you
+/// can set the [ThemeData.appBarTheme] property for your app
+/// using the [ThemeData] class.
+///
+/// Alternatively, you can set the [ThemeData.colorScheme] property
+/// to a custom [ColorScheme]. The [AppBar.backgroundColor] uses the
+/// [ColorScheme.surface].
```

Comment:
This isn't really advisable, i.e. it wouldn't really make sense to change ColorScheme.surface to tweak the color of the AppBar. I suppose we could just say as much, like "since many components depend on individual color scheme colors, applications typically customize them to produce an overall change in their app's appearance, rather than the appearance of an individual component.

Reply:
Sorry I just meant they could set the ColorScheme in a variety of ways (i.e. fromSeed) but then the actual surface property is what get's used ad the AppBar.backgroundColor